src/arbitrage.py

```diff
@@ -80,6 +83,48 @@ class API(object):
     def __init__(self, args=None):
         self.name = args
 
+    def get(self, stock):
+        exps = self.get_expiries(stock)
+        spot = self.get_spot_price(stock)
+        premiums = {}
+        for exp in exps:
+            premiums[exp] = self.get_option_premiums(stock, exp, spot)
+        return StockData(stock, exps, spot, premiums)
+
+    def get_expiries(self, stock):
+        if self.name == "tradier":
```

Comment:
It is better to have API as an interface and a separate TradierAPI class that implements this interface.

This allows us to program to an interface, which makes it easy to swap out APIs if we want to use a different one in the future.

Reply:
fixed